modules/Microsoft.Sql/servers/databases/deploy.bicep

```diff
@@ -47,8 +47,8 @@ param highAvailabilityReplicaCount int = 0
 @description('Optional. Minimal capacity that database will always have allocated.')
 param minCapacity string = ''
 
-@description('Optional. Time in minutes after which database is automatically paused.')
-param autoPauseDelay string = ''
+@description('Optional. Time in minutes after which database is automatically paused. A value of -1 means that automatic pause is disabled.')
```

Comment:
Is that documented like this? In the code below it doesn't seem we provide any corresponding logic ourselves (e.g., `autoPauseDelay: autoPauseDelay != -1 ? autoPauseDelay : null`)

Reply:
That comes from the corresponding Bicep resource reference description